qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -135,3 +176,19 @@ helm upgrade scf ${CAP_DIRECTORY}/helm/cf${CAP_CHART}/ \
 
 # Wait for CF namespace
 wait_for_namespace "${CF_NAMESPACE}"
+# While the background app monitoring job is running, *and* the app isn't yet ready, sleep
+while jobs %% &>/dev/null && ! tail -1 ${monitor_file} | grep -q "200 OK"; do
+  sleep 1
+done
+
+# If we get here because the app is ready, monitor_url will still be running in the background
+# Kill it, so we don't get any messages about the app becoming unreachable at the end
+if jobs %% &>/dev/null; then
+  echo "Terminating app monitoring background job"
+  kill %1
+fi
+
+echo "Results of app monitoring:"
```

Comment:
(suggestion) 
i think we can add one more echo line here:
echo "Total seconds : App State"